microcosm_pubsub/handlers/uri_handler.py

```diff
@@ -60,11 +61,13 @@ def __init__(
         retry_nack_timeout=1,
         resource_nack_timeout=1,
         resource_cache_enabled=True,
+        resource_cache_ttl=DEFAULT_RESOURCE_CACHE_TTL,
```

Comment:
@clatko-globe FYI changed this to be a simple constructor argument rather than a `graph.resource_cache` config parameter.. thinking is:
a. simpler signature here without having to access some other component's config block
b. in particular this lets us tune resource cache ttl *per handler*, which could be useful.